manifests/package/install.pp

```diff
@@ -7,21 +7,17 @@
 #
 # * Justin Lambert <mailto:jlambert@letsevenup.com>
 #
-class artifactory::install {
+class artifactory::package::install inherits artifactory::params {
 
   if $caller_module_name != $module_name {
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  File {
-    require => Package['artifactory'],
```

Comment:
I don't really remember removing this, but I can imagine why I did it. I really dislike this kind of monkey-patching of resource types, because it causes a lot of trouble sometimes. Can put it back in, no problem.